packages/js/product-editor/src/components/schedule-sale-pricing-block/edit.tsx

```diff
@@ -0,0 +1,149 @@
+/**
```

Comment:
Very nitpicky and up for debate, but should we simplify the name of this component folder to `schedule-sale`?

I think `pricing` and `block` don't add too much context and create more reading, though I could see the need to differentiate blocks from "regular" components (but in that case I might favor a separate `blocks/` folder 🤔 ).

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/37567/commits/fbaf580d2b704c9ae8b5e5b3ae096ce60fba6731